superset-frontend/src/views/CRUD/data/dataset/DatasetEditor/DatasetPanel/DatasetPanel.tsx

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
I would name all of these files index.tsx so you don't have the repetition at the end.

Reply:
thank you, changed.